src/java.base/share/classes/java/io/PushbackReader.java

```diff
@@ -254,15 +254,7 @@ public void close() throws IOException {
     }
 
     /**
-     * Skips characters.  This method will block until some characters are
-     * available, an I/O error occurs, or the end of the stream is reached.
-     *
-     * @param     n  The number of characters to skip
-     *
-     * @return    The number of characters actually skipped
-     *
-     * @throws    IllegalArgumentException  If {@code n} is negative.
-     * @throws    IOException  If an I/O error occurs
+     * {@inheritDoc}
```

Comment:
This appears to make the skip method disappear from the PushbackReader javadoc
and only show up as a method declared in FilterReader.

Reply:
This is intended and expected.